climpred/bootstrap.py

```diff
@@ -267,6 +271,8 @@ def bootstrap_compute(
           https://doi.org/10/f4jjvf.
 
     """
+    if dim is None:
```

Comment:
This should only be done if `dim` defaults to None. You can drop these two lines and just have `bootstrap_perfect_model` default to passing ['init', 'member']. You only do this to avoid having a default keyword be a list, but here the default keyword is `init`.

Reply:
this should belong to `compute_pm`